akka-docs/src/test/java/jdocs/actor/ActorDocTest.java

```diff
@@ -192,9 +192,7 @@ public Receive createReceive() {
      *         (e.g. calling `.withDispatcher()` on it)
      */
     static Props props(Integer magicNumber) {
-      // You need to specify the actual type of the returned actor
-      // since Java 8 lambdas have some runtime type information erased
-      return Props.create(DemoActor.class, () -> new DemoActor(magicNumber));
+      return Props.create(DemoActor.class, magicNumber);
```

Comment:
While this is shorter, I think I like the 'old' way a bit better here because it will cause a compiler error when the `DemoActor` constructor taking 1 integer is removed, instead of failing at run time. WDYT?

Reply:
ticket stated (and I checked)that documentation mention `create` with attributes as "best way". Now I can see that this was old docs.

Reply:
Ah yes, `Props.create(DemoActor.class, () -> new DemoActor(magicNumber))` is safer against refactoring, but `Props.create(DemoActor.class, magicNumber)` is safer against accidentally closing over (mutable) state... :+1: then :)